pkg/ocm/helpers.go

```diff
@@ -1039,3 +1040,30 @@ func ValidateBalancingIgnoredLabels(val interface{}) error {
 
 	return nil
 }
+
+func ValidateClaimValidationRules(input interface{}) error {
+	var idRE = regexp.MustCompile(`^[0-9a-z]+([:][0-9a-z]+)`)
+	var inputRules []string
+	inputType := reflect.TypeOf(input).Kind()
+	switch inputType {
+	case reflect.String:
+		if input.(string) == "" {
+			return nil
+		}
+		inputRules = strings.Split(input.(string), ",")
+		for _, inputRule := range inputRules {
+			if !idRE.MatchString(inputRule) {
+				return fmt.Errorf("invalid identifier '%s' for 'claim validation rule. '"+
+					"Should be in a <claim>:<required_value> format.", inputRule)
+			}
+		}
+	case reflect.Slice:
+		if reflect.TypeOf(input).Elem().Kind() != reflect.String {
+			return fmt.Errorf("unable to verify claim validation rules, incompatible type, expected slice of string got: '%s'",
```

Comment:
Do we also need to iterate through each entry in the slice and verify the format of the input rule?

Reply:
yeah, so the format will be claim:value, (optional more) claim:value... We'd need to verify that it's following this format.